bemani/backend/iidx/pendual.py

```diff
@@ -889,46 +900,40 @@ def handle_IIDX22pc_request(self, request: Node) -> Optional[Node]:
 
             limit = Node.void('limit')
             root.add_child(limit)
-            limit.set_attribute('phase', '24')
+            limit.set_attribute('phase', '21')
 
             # See if we configured event overrides
             if self.machine_joined_arcade():
                 game_config = self.get_game_config()
                 timeshift_override = game_config.get_int('cycle_config')
-                omni_events = game_config.get_bool('omnimix_events_enabled')
+                ccxbm_on = game_config.get_bool('ccxbm_enable')
             else:
                 # If we aren't in an arcade, we turn off events
                 timeshift_override = 0
-                omni_events = False
-
-            if self.omnimix and (not omni_events):
```

Comment:
Second comment on this, why remove this?

Reply:
See explanation above in the first comment.